fares-data-build-tool/fdbt-site
# test: improve testing coverage

# Description

-   This PR improves testing coverage on the site.

# Testing instructions

-   Pull down this branch and run the site locally. Check that the new tests are valid and that they increase coverage.

# Type of change


# Checklist:
